vitest.shared.ts

```diff
@@ -17,5 +17,8 @@ export default defineConfig({
 			useAtomics: true,
 		},
 		restoreMocks: true,
+		// A lot of the fixture tests are extremely flaky because of the dev registry
+		// Retry tests by default so that only real errors are reported
+		retry: 2,
```

Comment:
I'm not against this change, but was wondering if adding these retries won't increase our build times even more

Reply:
This only affects packages which run tests with vitest, which are all so fast it doesn't make much of a difference (it excludes wrangler and miniflare, for instance). I think the extra robustness should make up for potentially longer build times